pkg/cover/instrument.go

```diff
@@ -127,12 +153,15 @@ func registerHandlers() {
 		log.Fatalf("register address %v failed, err: %v, response: %v", profileAddr, err, string(resp))
 	}
 	go genProfileAddr(host)
-
+	//clear and profile is mutex,
+	var lock atomic.Value
 	mux := http.NewServeMux()
 	// Coverage reports the current code coverage as a fraction in the range [0, 1].
 	// If coverage is not enabled, Coverage returns 0.
 	mux.HandleFunc("/v1/cover/coverage", func(w http.ResponseWriter, r *http.Request) {
+		lock.Store(false)
```

Comment:
it's easier to understand that the `lock` is true when it's doing locking. and false when released